packages/p2p/src/components/advertiser-page/advertiser-page.jsx

```diff
@@ -51,7 +54,17 @@ const AdvertiserPage = () => {
     }
 
     return (
-        <div className='advertiser-page'>
+        <div
+            className={classNames('advertiser-page', {
+                'advertiser-page--no-scroll': !!is_blocked,
```

Comment:
Made advertiser page non-scrollable is block user overlay is shown as per QA requirements